### PR TITLE
Update androidx.core, firebase, activityVersion, pagingVersion

### PR DESCRIPTION
**Pull request by GeorgCantor**
Updated some libraries versions

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ buildscript {
         compileSdkVersion = 30
         targetSdkVersion = 30
 
-        activityVersion = '1.2.0-rc01'
+        activityVersion = '1.3.0-alpha02'
```
Comment (ianhanniballake) [inline build.gradle:22]: This should be `1.2.0`. Muzei doesn't rely on anything in Activity 1.3 (which is only relevant to Compose apps anyways).

File: muzei-api/build.gradle [hunk 2]
```diff
@@ -36,7 +36,7 @@ allprojects { ext."signing.secretKeyRingFile" = props["signing.secretKeyRingFile
 
 dependencies {
     api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
-    api "androidx.core:core:1.1.0"
+    api "androidx.core:core-ktx:1.3.2"
```
Comment (ianhanniballake) [inline muzei-api/build.gradle:39]: This is intentionally `core:1.1.0` so that to avoid forcing unnecessary updates to Core in apps that depend on the `muzei-api` (Muzei API uses the minimal version of Core that it needs). Please revert this change.

**Review by ianhanniballake — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 3]
```diff
@@ -19,7 +19,7 @@ buildscript {
         compileSdkVersion = 30
         targetSdkVersion = 30
 
-        activityVersion = '1.3.0-alpha02'
+        activityVersion = '1.2.0-rc01'
```
Comment (ianhanniballake) [inline build.gradle]: The revert here wasn't what I had asked for - I asked for `1.2.0`. We still want to upgrade from the RC to stable release, there's just no reason to go up to Activity 1.3

**Review by ianhanniballake — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 3 above)
Comment (GeorgCantor) [reply build.gradle]: Ok. Sorry

**Review by ianhanniballake — APPROVED** [review]
